_posts/2015-11-09-scrum-basics-2.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Scrum Basics Series - Part 2: Scrum Roles"
+date: "2015-11-09 07:23:00"
+icon: university
+categories: agile
+authors: ["Anja B."]
+---
+
+Welcome to part two of our Scrum Basics Series.
+This part deals with the three different roles involved in the Scrum process.
+
+One thing to clarify right from the beginning: All three roles together form one Scrum Team.
+All are equal and neither the Product Owner (PO) nor the Scrum Master (SM) are superiors to the Development Team.
+To function well they need to work closely together and, at best, also sit in one room.
+They need to understand that they are ONE team, fighting for the same goal: to release the best possible product for the end-customer.
+
+# Development Team
+
+{% image blog-scrum2-devteam.jpg 50% left %}{% endimage %}
+
+A typical Development Team consists of 7 members +/-2 with cross-functional abilities.
+They should organise themselves and decide on their own, how tickets will be implemented within a sprint.
+It’s also the Development Team's decision, how many tickets they decide to solve within a sprint.
+This freedom comes with the duty to commit completely to the goal to deliver all the planned tickets within a sprint.
+
+To being able to know how many tickets they are usually able to solve within a sprint, the team estimates all tickets by themselves.
+Nobody else is allowed to give estimates for tickets but the Development Team as they will solve those tickets in the end.
+During the sprint, the Development Team also has the duty to track their progress.
+That means, they keep the Sprint/Scrum Board clean and move the cards across the board according to their progress.
+Everyone in the team, including PO and SM, should always know the status of each ticket within a sprint.
+
+The most important task for the Development Team, also in Scrum, is to deliver.
+Just in Scrum they can tell themselves, how much time they need to deliver something, which gives them the freedom to achieve the big goal of the Scrum Framework: to deliver quality products.
+
+# Product Owner
+
+{% image blog-scrum2-productowner.jpg 30% left %}{% endimage %}
+
+The Product Owner is a very important role in Scrum.
+I’ve heard trainers say that a Scrum Team can rise or fall with their PO.
+The speciality in Scrum is, that the PO is not the boss of the Development Team, but part of the team.
+This is not easy, especially for POs who have worked as Project Managers in the past.
+In Scrum they are a lateral leader.
+
+However, they do decide what the team is going to develop next.
+They are setting up the Product Backlog and decide the business value of each ticket and with that the ticket order.
```

Comment:
@BirgitBader @JessTinker indeed "business value" is a basic scrum term, so it might be sense-full to explain that:

> ... business value (e.g. new features, better maintainability, bug fixes, etc.) ...